workshop_R.Rmd

```diff
@@ -507,8 +586,16 @@ The above code gets us the correct geographic level and region name, but not reg
 
 The [dfe-analytical-services](https://github.com/dfe-analytical-services) area on GitHub holds all publicly available code from DfE. The code behind the [EES data screener](https://rsconnect/rsc/dfe-published-data-qa/) is available here, and [the screener repository](https://github.com/dfe-analytical-services/dfe-published-data-qa) contains a [folder full of geographic lookup tables](https://github.com/dfe-analytical-services/dfe-published-data-qa/tree/master/data)! 
```

Comment:
"area on GitHub holds all the publicly available code" - maybe change this to holds a range of useful publicly available code? The data science team have their own GitHub repo with some DfE code.